### ISSUE TITLE
Issue with class attribute in 11.1.0

### ISSUE THREAD

**Issue by vaindil** [czeckd/angular-svg-icon#118]
I just upgraded our project to 11.1.0, and it now fails at runtime. We have a lot of elements that look like this:

```
<svg-icon name="whatever_name" class="some-class another-class"></svg-icon>
```

This worked correctly in previous versions, but now it fails with this error:

> DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('some-class another-class') contains HTML space characters, which are not valid in tokens.

Looks like this is happening because of the change in #117, which is using [`Renderer2.addClass`](https://angular.io/api/core/Renderer2#addClass). That method only supports providing one class at a time, not a string of multiple.

**Comment by czeckd** [comment]
Thank you for the bug report. This should be now resolved in v11.1.1.

**Comment by vaindil** [comment]
Sorry for the delay, looks like it's working. Thank you!
